roles/node/tasks/config-chains.yml

```diff
@@ -17,3 +17,12 @@
     group: "{{ avalanchego_group }}"
   loop: "{{ avalanchego_chains_configs | dict2items }}"
   notify: Restart avalanchego
+
+- name: Generate chains upgrade.json
```

Comment:
For safety, we need to enrich the `Create chains conf directories` task above with the `avalanchego_chains_updates.keys()` in case a chain has upgrades but no custom config. I think you can just use the `union` filter to combine the lists.

Reply:
ahh, good point.

Reply:
a one-liner on the loop line is not working withj2 templating so I added vars
but I believe it is better for readability as well

Reply:
This works